includes/customizer-functions.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Register Customizer controls which add Elementor deeplinks
+ *
+ * @return void
+ */
+add_action( 'customize_register', 'hello_customizer_register' );
+function hello_customizer_register( $wp_customize ) {
+
+	require get_template_directory() . '/includes/settings/hello-elementor-customizer-upsell.php';
```

Comment:
When the customizer control file is moved, PHP Storm should refactor it for you but make sure it's good and works plz

Reply:
Done.